word_occupancy/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include <string>
+#include "lib.h"
+
+int main()
+{
+    std::string given_string;
+    std::getline(std::cin, given_string);
```

Comment:
modify so that we could pass file as argument instead of STDIN

Reply:
++